### ISSUE TITLE
Fix Building / Testing in Docker instructions

### ISSUE THREAD

**Issue by cajun-rat** [advancedtelematic/aktualizr#101]
The instructions to build in Docker in the README.md don't currently work. We should fix this :)

```
docker build -t advancedtelematic/aktualizr .
docker run --rm -it advancedtelematic/aktualizr make
make: *** No targets specified and no makefile found.  Stop.
```

**Comment by cajun-rat** [comment]
I'm working on this

**Comment by cajun-rat** [comment]
Fixed in #a813df712a63821831da56ec712d62b88a27767d
